modules/ximgproc/samples/thinning.cpp

```diff
@@ -10,7 +10,7 @@ using namespace cv;
 
 int main()
 {
-    Mat img = imread("opencv-logo.png", IMREAD_COLOR);
+    Mat img = imread("../data/opencv-logo.png", IMREAD_REDUCED_COLOR_2);
```

Comment:
I am not convinced that changes like this work. If the samples get build, the data will not be one folder up. It is only like this in the case of the source files. Thus I suggest to undo this change.